e2e_testing/torchscript/rng.py

```diff
@@ -90,17 +89,34 @@ def __init__(self):
     @export
     @annotate_args([
         None,
-        ([-1, -1, -1], torch.float32, True),
+        ([-1, -1, -1], torch.float64, True),
+        ([-1, -1, -1], torch.float64, True),
+        ([-1, -1, -1], torch.float64, True),
     ])
-    def forward(self, a):
-        x = torch.bernoulli(a)
-        mean = torch.mean(x)
-        std = torch.std(x)
+    def forward(self, x, y, z):
+        a = torch.bernoulli(x)
+        b = torch.bernoulli(y)
+        c = torch.bernoulli(z)
+        mean = torch.cat([
+            torch.flatten(torch.mean(a)),
+            torch.flatten(torch.mean(b)),
+            torch.flatten(torch.mean(c))
+        ])
+        std = torch.cat([
+            torch.flatten(torch.std(a)),
+            torch.flatten(torch.std(b)),
+            torch.flatten(torch.std(c))
+        ])
         return  mean, std
 
 @register_test_case(module_factory=lambda: BernoulliModule())
 def BernoulliModule_basic(module, tu: TestUtils):
-    module.forward(tu.rand(256, 512, 64))
+    module.forward(
+        tu.rand(256, 512, 8).double(),
+        tu.rand(512, 1024, 4).double(),
+        tu.rand(512, 256, 4).double())
```

Comment:
Please also test extreme cases like `torch.zeros` and `torch.ones` or `torch.ones() * 0.7` to test different distributions.

Reply:
Done. Thanks :)